postgres-appliance/docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+version: "3.5"
+services:
+  spilo:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        # set to true if you want to install perl and plperl packages into image
+        WITH_PERL: 'false'
+        PGVERSION: '13'
```

Comment:
Yes they are backward compatible [1] but I would still like to use 13.3 version (same as we use in production). 

I will check how it can be used in dockerfile

[1] https://www.postgresql.org/docs/13/upgrading.html

Reply:
UPDATE - I tried to use 13.3 and building image failed due to missing packages. I think I need to use 13. 

After successful building of image, I will use new postgres 13.x version in our patagona-api and see it everything works as expected.

```
Building dependency tree...
Reading state information...
E: Unable to locate package postgresql-contrib-13.3
E: Couldn't find any package by glob 'postgresql-contrib-13.3'
E: Couldn't find any package by regex 'postgresql-contrib-13.3'
E: Unable to locate package postgresql-plpython3-13.3
E: Couldn't find any package by glob 'postgresql-plpython3-13.3'
E: Couldn't find any package by regex 'postgresql-plpython3-13.3'
E: Unable to locate package postgresql-server-dev-13.3
E: Couldn't find any package by glob 'postgresql-server-dev-13.3'
E: Couldn't find any package by regex 'postgresql-server-dev-13.3'
E: Unable to locate package postgresql-13.3-cron
E: Couldn't find any package by glob 'postgresql-13.3-cron'
E: Couldn't find any package by regex 'postgresql-13.3-cron'
E: Unable to locate package postgresql-13.3-pgq3
E: Couldn't find any package by glob 'postgresql-13.3-pgq3'
E: Couldn't find any package by regex 'postgresql-13.3-pgq3'
E: Unable to locate package postgresql-13.3-pg-stat-kcache
E: Couldn't find any package by glob 'postgresql-13.3-pg-stat-kcache'
```